src/components/YoutubeNav/YoutubeNav.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Nav from './YoutubeNav.component';
+
+describe('Nav elements', () => {
+  test('must display navbar', () => {
+    render(<Nav />);
+    const icons = screen.getAllByRole('navigation');
+    expect(icons).toHaveLength(1);
```

Comment:
I think `toBeInTheDocument` could be a more appropriate assertion for this.

Reply:
Unless there are more than one icon with the role navigation, and I would use getByRole without the `all`.